styles/styles.css

```diff
@@ -105,6 +105,27 @@
   --font-weight-semibold: 600;
   --font-weight-bold: 700;
   --font-weight-bolder: 800;
+
+  /* margin and fine spaces */
+  --space-1px: 0.006rem;
```

Comment:
we should go for full value for all.

Reply:
I took values from here - https://nekocalc.com/px-to-rem-converter 
I think 3 decimal places are sufficient since <1 px is too small and any further additions would be unnoticeable